sideboard/lib/_threads.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import unicode_literals
 import time
 import heapq
+import prctl
```

Comment:
From Dom in Slack, this is because the module isn't installed and re-running puppet (`cd ~/uber/puppet && ./apply_node.sh localhost`) will make it work.